controllers/apply/under10k/fields/your-idea.js

```diff
@@ -99,9 +99,39 @@ module.exports = {
         const minWords = 50;
         const maxWords = 150;
 
-        function guidanceText() {
-            const prioritiesCOVID19 = localise({
-                en: `<ol>
+        function prioritiesCOVID19() {
+            if (flags.enableGovCOVIDUpdates) {
```

Comment:
Behind a flag as this priorities don't come into effect until we launch new funding